### DISCUSSION TITLE
i installed both github and store version, but they did not merge, how can i fix this?

### DISCUSSION THREAD

**Discussion by ghost** [t1m0thyj/WinDynamicDesktop#385] [Q&A] [answered]
i installed both github and store version, but they did not merge, how can i fix this?

**Comment by t1m0thyj** [comment, marked as answer]
They won't merge because they are installed in 2 separate locations. Microsoft Store apps are typically installed in the location `C:\Users\<user>\AppData\Local\Packages`, while the desktop app published on GitHub is installed wherever you choose in the installer.

You can just uninstall the version you don't want. If you have themes you want to copy from one installation to another, see [here](https://github.com/t1m0thyj/WinDynamicDesktop/wiki/Troubleshooting#finding-where-app-is-installed) to find where they are located.
